File: End2EndApp/EmployeeWebApi/Models/Employee.cs
namespace EmployeeWebApi.Models
{
    public class Employee
    {
        public int empId { get; set; }
        public string empName { get; set; }
        public string empAddress { get; set; }
        public decimal empSalary { get; set; }

        public int deptId { get; set; }
    }
}

File: Angular/fnftraining/src/app/Models/employee.ts
//interface is simliar to struct of C. So if U want a udt that does not have method but only data, then U can create it as an interface. 
export interface Employee {
    empId : number,
    empName : string, 
    empAddress : string, 
    empSalary : number,
    empPic : string
}
